projects/demo/src/modules/pipes/filter-by-input/examples/2/index.ts

```diff
@@ -27,7 +28,7 @@ interface User {
     encapsulation,
     changeDetection,
 })
-export default class Example {
+export default class Example<T extends User> {
```

Comment:
In this case it could also be `T extends User = User`. Then it might not be necessary to add `as T[]`.

Reply:
<img width="829" alt="image" src="https://github.com/user-attachments/assets/94449fdb-5611-4e71-9d2d-5dc16418934f">

Reply:
Ok, nvm. Bad assumption 🙂